manifests/params.pp

```diff
@@ -310,13 +316,23 @@
   }
 
   $server_jvm_java_bin = '/usr/bin/java'
-  $server_jvm_min_heap_size = '2G'
-  $server_jvm_max_heap_size = '2G'
+  $server_jvm_min_heap_size = '4G'
+  $server_jvm_max_heap_size = '4G'
```

Comment:
Puppetserver 1.1.3 won't even run with less than 4G of RAM, I believe. I recall trying to stand up a very tiny box, only to be slapped down by saying it had a minimum required memory.